README.md

```diff
@@ -36,3 +36,41 @@ dependencies {
   compile "com.teamwizardry.librarianlib:librarianlib-1.12:4.0-SNAPSHOT:deobf"
 }
 ```
+
+## Features
+(note: this list was thrown together based on the package list, so it is not exhaustive)
+
+- reflection-based animation
+- annotation-based automatic block/item annotation
+- base capability implementation
+- base blocks/items/entities/fluids/multiparts 
+- base tile entities with automatic serialization
+- automatic model generation for everything under the sun (plants, fences, doors, you name it)
+- per-chunk data storage
+- per-world data storage
+- annotation-based config handler
+- a base container that simplifies shift-clicking to just be `transferRule().from(inventory1).to(inventory2).filter { some item filter }`
+- a simplified event bus for liblib's gui framework
+- a giant gui framework (from what I can tell, it's one of, if not the, most advanced ones in MC)
+- a data-driven guide book implementation using said gui framework
+- a massively reworked version of the giant gui framework
+- a hud handler using the gui framework, with gui components that are automatically positioned to match every minecraft hud element
+- a data-driven guide book implementation using said gui framework
+- a set of simple GUI components designed to closely emulate desktop UI, meaning the user doesn't have to guess at how controls might behave (e.g. button, toggle button, spinner button, etc)
+- a shit ton of small helpers
+- a shit ton of small kotlin-specific helpers
+- a bunch of math classes
+- a methodhandle helper for native-speed reflective method/field access
+- base network implementation with automatic packet serialization
+- a powerful but inefficent particle system
+- a new, more powerful, and hyper-efficient particle system (I've gotten 20k particles with world collision running on my potato)
+- an advanced reflection-based automatic serialization library
```

Comment:
This feature was kind of hinted at by other features (auto packet serde, auto TE serde, ...)
Maybe have this one higher up the list and reference it in the other features ?